fe/fe-core/src/main/java/com/starrocks/common/util/PulsarUtil.java

```diff
@@ -0,0 +1,167 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
new added file should use following license header

// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.